content/v1/csidriver/features/powerflex.md

```diff
@@ -287,6 +287,18 @@ allowedTopologies:
     - csi-vxflexos.dellemc.com
 ```
 
+**Note** : 
+  The NFS labels are automatically added by all drivers, assuming that NFS dependencies are configured by default. These dependencies come with the default Linux OS package from the node and the array supports NFS. For other protocols (iSCSI, FC, NVMe), all necessary connectivity checks between the initiator and target are performed before labels are added.
```

Comment:
Powerflex doesn't support iscsi, FC and Nvme yet.

Reply:
Also, for powerflex, this will be added if version is >=4. Please cross check this one.

Reply:
Updated